server/auth/auth.go

```diff
@@ -153,22 +153,29 @@ func (client *clientInfo) pruneBonds(lockTimeThresh int64) (pruned []*db.Bond, b
 		return
 	}
 
-	var n int
+	pruned = make([]*db.Bond, 0, len(client.bonds))
```

Comment:
We don't really need to pre-allocate `pruned`. It won't be used often, and when it is is will be a single bond pretty much every time.